app/api/unit_activity_sets_api.rb

```diff
@@ -0,0 +1,82 @@
+require 'grape'
+
+module Api
+  class UnitActivitySetsApi < Grape::API
+    helpers AuthenticationHelpers
+    helpers AuthorisationHelpers
+
+    before do
+      authenticated?
+    end
+
+    desc 'Add an unit activity set'
+    params do
+      requires :activity_type_id, type: Integer, desc: 'The id of the activity type'
```

Comment:
Should this have a name/abbreviation? eg Prac01

Reply:
Isn't prac01 an activity?
Unit activity set will have many activities?

Reply:
It could be an activity set - for example Prac01 and Prac02 with activities Prac01-1, Prac01-2, Prac02-1, Prac02-2 etc.

We need some way to name the activities. Just an id and type is insufficient. What if there are two Prac activity sets in a unit?